README.md

```diff
@@ -1,20 +1,67 @@
+> [!IMPORTANT]
+> ThreadSharp is limited by the capabilities of the [Threads public API](https://developers.facebook.com/docs/threads/) and so not all features that you enjoy in the Threads app are available.
+
 ![ThreadSharp banner](https://github.com/itsWindows11/ThreadSharp/blob/main/assets/banner.png?raw=true)
+<h4 align="center"> A C# API wrapper for the Threads API.</h4>
+<p align="center">
+  <a style="text-decoration:none" href="https://github.com/itsWindows11/ThreadSharp/actions/workflows/nuget.yml">
+    <img src="https://github.com/itsWindows11/ThreadSharp/actions/workflows/nuget.yml/badge.svg" alt="CI Status" /></a>
+  <a style="text-decoration:none" href="https://discord.com/channels/714581497222398064/913496895006072843">
+    <img src="https://img.shields.io/badge/discuss-on%20discord-5865F2" alt="Discord" /></a>
```

Comment:
This Discord server isn't a *strictly* official way to discuss the library. I might use a separate Discord server for the project.